west.yml

```diff
@@ -0,0 +1,35 @@
+west:
```

Comment:
file TBR

Reply:
If the maintainers of MCUBoot will allow tight coupling with fw-nrfconnect-nrf I'm fine with that and I'll change this PR

Reply:
@nvlsianpu: This is now updated